apps/condo/domains/billing/components/OnBoarding/index.tsx

```diff
@@ -43,21 +47,36 @@ export const BillingOnboardingPage: React.FC<BillingOnboardingPageProps> = ({ on
     const router = useRouter()
     const [welcomeShown, setWelcomeShown] = useState(false)
 
-    const [currentStep] = useOnboardingProgress()
+    useEffect(() => {
+        setWelcomeShown(Boolean(localStorage.getItem(MODAL_STORAGE_KEY)))
```

Comment:
Will it working with SSR?

Reply:
useEffect is only triggered on a client as far as I remember, so it will.

Here's a reference in our code btw: https://github.com/open-condo-software/condo/blob/master/apps/condo/domains/common/components/LayoutContext.tsx#L44

Reply:
First call of useEffect will proceed at server render. There is reference to similar [problem](https://stackoverflow.com/questions/73853069/solve-referenceerror-localstorage-is-not-defined-in-next-js)